server/README.md

```diff
@@ -65,6 +65,12 @@ gem install bundler
 gem install rails
 ```
 
+Install libpq-dev package using
```

Comment:
```suggestion
(If you are using Ubuntu or Debian Linux:) Install the development package for the PostgreSQL database `libpq-dev` package using
```

Perhaps this helps people on Windows know that they shouldn't do anything?

Reply:
Yes, mentioning OS will help people to know that they don't have to use this command on Windows. But, I am not very sure if they required to install any package on windows system :( .
I followed [this](https://stackoverflow.com/a/6040822) to resolve my issue.